client-app/shared/catalog/components/product-properties.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div v-for="property in grouped" :key="property.name" class="flex mb-4 space-x-3">
+    <div class="flex w-1/2 space-x-3">
+      <div class="flex-shrink text-gray-500">
+        {{ property?.name }}
+      </div>
+      <div class="flex-1 border-b border-grey-100 border-dotted h-5"></div>
+    </div>
+    <div class="font-bold w-1/2 flex flex-col text-left">
+      {{ property.values }}
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import _ from "lodash";
+import { computed, PropType } from "vue";
+import { Maybe, Property } from "@core/api/graphql/types";
+
+const props = defineProps({
+  properties: {
+    type: Object as PropType<Array<Maybe<Property>>>,
+    default: () => new Array<Maybe<Property>>(),
+  },
+});
+
+const grouped = computed(() => {
```

Comment:
Please consider moving out this logic into one place (specialized utility class in the product domain will be a good solution)